packages/gensx/package.json

```diff
@@ -51,12 +51,15 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.2.3",
     "@rollup/plugin-typescript": "^11.1.6",
+    "@gensx/openai": "workspace:*",
```

Comment:
hmm. I think this is a circular dependency.

Reply:
oh yeah not sure why that's in there; we don't need it.